gavel.py

```diff
@@ -14,220 +13,25 @@
     session,
     url_for,
 )
-from flask.ext.sqlalchemy import SQLAlchemy
-from sqlalchemy.orm.exc import NoResultFound
-from sqlalchemy.exc import IntegrityError
-from sqlalchemy.sql.expression import func, desc
-from datetime import datetime
-from numpy.random import (
-    choice,
-    random,
-    shuffle,
-)
-import crowd_bt
-from functools import wraps
-import csv
-import io
+from models import IntegrityError, Decision, Annotator, Item, db, ignore_table
```

Comment:
Why is `IntegrityError` imported from `models` when it's defined in `sqlalchemy.exc`?